experimental/PropertyDDS/packages/property-dds/src/propertyTree.ts

```diff
@@ -791,6 +796,9 @@ export class SharedPropertyTree extends SharedObject {
 
 	getRebasedChanges(startGuid: string, endGuid?: string) {
 		const startIndex = findIndex(this.remoteChanges, (c) => c.guid === startGuid);
+		if (startIndex === -1 && startGuid !== "") {
+			console.error("Unknown start GUID specified.");
```

Comment:
Note: this originally was an exception, but I changed it to console.error for now. As long as there are still clients in a collaboration session that are using the old version of the DDS, throwing an error here makes the situation worsed. Previously, if a change with a start GUID outside of the collaboration window was sent, it would only trigger an error if this change actually caused a rebase conflict. Without any other conflicting changes it would go through just fine. With this new error message, it would break every time, if we throw here.

So to avoid making the situation worse until everybody has migrated, I am only printing a message here, but not throwing.

Reply:
FYI - I added a short TODO comment referencing this PR.